app/controllers/hosts_controller.rb

```diff
@@ -918,4 +921,8 @@ def host_attributes_for_templates(host)
        host_attributes.include?(k) && !k.end_with?('_ids')
     end.except(:host_parameters_attributes)
   end
+
+  def csv_columns
+    [:id, :name, :operatingsystem, :environment, :model, :hostgroup, :last_report]
```

Comment:
What do you think about removing the ID column? I'm not sure if its required (nor its visible in the current table).